docs/web-console-guide/master.adoc

```diff
@@ -128,6 +128,8 @@ Assessing applications refers to estimating the risks and costs involved in prep
 Analyzing applications refers to using rules to determine which specific lines in an application must be modified before the application can be migrated or modernized.
 
 include::topics/mta-web-assessing-apps.adoc[leveloffset=+2]
+include::topics/mta-web-assessment-module.adoc[leveloffset=+3]
+include::topics/mta-web-archetypes.adoc[leveloffset=+3]
```

Comment:
Did you see the build error in the preview?

![image](https://github.com/windup/windup-documentation/assets/60698649/85d2c8b6-d613-4f09-a89c-c3b638b75b36)